packages/connect/src/utils/assets.native.ts

```diff
@@ -1,3 +1,33 @@
+import { HttpRequestType, HttpRequestReturnType, HttpRequestOptions } from './assetsTypes';
 import { tryLocalAssetRequire } from './assetUtils';
 
-export const httpRequest = (url: string, _type: string): any => tryLocalAssetRequire(url);
+export function httpRequest<T extends HttpRequestType>(
```

Comment:
This might close this issue too, right? https://github.com/trezor/trezor-suite/issues/13271

Reply:
Yes, I think it should, be we should test it including FW translations.